### PR TITLE
Refactored code and improved logging 

### PR DESCRIPTION
**Pull request by simsine**
The code is refactored to be more tidy and easy to  develop.
 Also improved logging to make it clear where the bot messages are sent.

### CODE REVIEW

**Comment by simsine** [comment]
Production enviornment is also updated with new env variable

File: src/config.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default {
+	clientURL: "https://chat.fribyte.no",
```
Comment (mathiash98) [inline src/config.ts:2]:
```suggestion
	clientURL: "https://chat.fribyte.no",
	get channelName() {
		return process.env.NODE_ENV === "production" ? "dugnad" : "bot-test";
	},
	get isProd() {
		return process.env.NODE_ENV === "production";
	},
	get token() {
		return process.env.TOKEN ?? throw new Exception("Token is not defined in environment. Add 'TOKEN=<YOUR TOKEN>' to the file '.env' in root folder.");
	}
```

I like to put all config stuff and env variables accessible in the config object. And then only use config all other places in the code.

Comment (simsine) [reply src/config.ts:2]: Me likey. 
Also never seen the get syntax before, js is one hell of a language

**Review by simsine — COMMENTED** [review]
:+1:

**Review by mathiash98 — APPROVED** [review]

**Pull request merged**